forms-flow-api/src/formsflow_api/resources/filter.py

```diff
@@ -118,6 +118,45 @@
     },
 )
 
+filter_preference_model = API.model(
+    "FilterPreferenceItem",
+    {
+        "filterId": fields.Integer(description="ID of the filter"),
+        "sortOrder": fields.Integer(description="Sort order for the filter"),
+        "hide": fields.Boolean(description="Whether the filter is hidden"),
+    },
+)
+filter_preference_request_model = API.model(
+    "FilterPreferenceRequestModel",
+    {
+        "body": fields.List(
+            fields.Nested(filter_preference_model),
```

Comment:
The filter request payload in the below format right?
[
  {"filterId": 1, "sortOrder": 1},
  {"filterId": 2, "sortOrder": 2}
] 
then you can give `@API.doc(body=[filter_preference_model])`